src/ServicePulse.Host/app/js/services/services.service-control.js

```diff
@@ -21,6 +21,13 @@
             });
         }
 
+        function getLicenseDetails() {
```

Comment:
Agree with moving this to it's own service. There may be times where the license comes from somewhere different to the SC endpoint in the future.